src/server/readpool/config.rs

```diff
@@ -134,13 +80,113 @@ impl Config {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Default)]
+#[serde(default)]
+#[serde(rename_all = "kebab-case")]
+/// A helper struct to handle deserialization default values.
+pub struct SerdeConfigHelper {
+    pub high_concurrency: Option<usize>,
```

Comment:
This is a struct for holding deserialized values from the configuration. User may omit some of the field in the config file, so that the corresponding field becomes `None`. We cannot use the `default()` here, because of serde's limitation: it does not support specifying different `default()` for the same type in different places (note: our coprocessor and storage uses different default values). That's why this PR is proposed in fact.

Reply:
The core idea of this PR is that, we provide the same default values for serde (fields set to None) and we derive this struct later for specialized purpose, i.e. coprocessor / storage. During this step, we need to know exactly whether a field is specified in the config or not. `Option` is very suitable for this purpose. Any way, this PR is a workaround for serde's "by design" feature.